x/auth/types/account_test.go

```diff
@@ -427,7 +427,7 @@ func TestTrackUndelegationDelVestingAcc(t *testing.T) {
 	now := tmtime.Now()
 	endTime := now.Add(24 * time.Hour)
 
-	_, _, addr := keyPubAddr()
+	_, _, addr := KeyTestPubAddr()
```

Comment:
Does this need to be exposed?

Reply:
if we don't want duplicate code it does need to be exposed... ideally this will move out into a common testing-helpers root level package at some point!